src/test/consensus/Consensus_test.cpp

```diff
@@ -295,6 +295,43 @@ class Consensus_test : public beast::unit_test::suite
 
         }
 
+        // Additional test engineered to switch LCL during the establish phase.
```

Comment:
Or more accurately, an assert should fire. But that only crashes in debug builds by default.